packages/flutter/lib/src/material/dialog.dart

```diff
@@ -217,6 +217,37 @@ class AlertDialog extends StatelessWidget {
     this.semanticLabel,
     this.shape,
   }) : assert(contentPadding != null),
+       _scrollable = false,
+       super(key: key);
+
+  /// Creates an alert dialog with scrollable [title] and [content].
+  ///
+  /// In contrast with a typical alert dialog, this configuration
+  /// is used when the [title] and [content] is expected to overflow.
+  /// Both [title] and [content] is wrapped in a scroll view, allowing
+  /// all overflowed content to be visible while still showing the button
+  /// bar.
+  ///
+  /// Typically used in conjunction with [showDialog].
+  ///
+  /// The [contentPadding] must not be null. The [titlePadding] defaults to
+  /// null, which implies a default that depends on the values of the other
+  /// properties. See the documentation of [titlePadding] for details.
+  const AlertDialog.scrollable({
```

Comment:
In general it looks like a good feature. However, why not just add a `scrollable` param to the main 'AlertDialog' constructor rather than creating an whole new constructor for just this setting?

Reply:
Done